src/PackageDependencyViewExtension/PackageDependencyView.xaml.cs

```diff
@@ -38,6 +38,7 @@ internal void OnWorkspaceCleared(IWorkspaceModel obj)
         {
             if (obj is WorkspaceModel)
             {
+                currentWorkspace.PropertyChanged -= OnWorkspacePropertyChanged;
```

Comment:
this seems like an odd place to do this -couldn't the workspace be cleared for some other reason... and if you clear the workspace but then don't add the the subscription back then you're not watching the workspace any longer...

I would have imagined this would be done in the `onCurrentWorkspaceChanged` handler - you cache the current workspace, unsubscribe the handler, then get the new current workspace and then subscribe?

Does that not work?